version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "15.9.11-SNAPSHOT"
```

Comment:
Should this be in this PR?

Reply:
I _think_ that update happens direct to master when the release is done?

Reply:
Yeah - that's a mistake. Dirty workspace.
 Reverting.